.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+        - c4t
+        - dev
+env:
+  docker_image: "c4tplatform/camino-conduit"
+
+jobs:
+  build_docker:    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)